Formula/mdbm.rb

```diff
@@ -0,0 +1,25 @@
+class Mdbm < Formula
+  desc "Y! MDBM a very fast memory-mapped key/value store"
+  homepage "https://github.com/yahoo/mdbm/"
+  url "https://github.com/yahoo/mdbm/archive/v4.13.0.tar.gz"
+  sha256 "99cec32e02639048f96abf4475eb3f97fc669541560cd030992bab155f0cb7f8"
+
+  depends_on "coreutils" => :build
+  depends_on "cppunit" => :build
+  depends_on "make" => :build
+  depends_on "openssl"
```

Comment:
We now use OpenSSL 1.1:

```suggestion
  depends_on "openssl@1.1"
```

Reply:
added it, thx